### PR TITLE
[MRG] Teaser for video-ident

### PR DESCRIPTION
**Pull request by michelfernandez**
- Added new teaser for video-ident at home
- Changed AES-for-Biz teaser to 2 column module

### CODE REVIEW

File: site/content/english/_index.md [hunk 1]
```diff
@@ -219,9 +219,34 @@ with legal form requirements or high liability risk.
...
+or in a few days the whole organization
+===============
+{{% /markdown %}}
+{{< /content >}}
+
+{{< content top=4 bottom=24 width=narrow display=flex >}}
+{{< step >}}
+{{< picture "video-ident" 250 "A magnifying glass enlarges the image of a man in a passport." >}}
+{{% markdown %}}
+**Online identifikation**
```
Comment (betatim) [inline site/content/english/_index.md:234]:
```suggestion
**Online identification**
```

File: site/content/english/identification.md [hunk 2]
```diff
@@ -170,16 +172,16 @@ Appropriate if new QES signers are constantly being added.
...
+  "Enable QES now"
+  "https://www.videoident.me/"
+  "_blank"
   "outline"
 >}}
 
 {{< picture id-direct 210 "A man is sitting in front of a laptop, above him is a sotware window, where a woman stands out with a big magnifying glass and checks an identity card." >}}
 
-You come to us or one of our partners for identification.
+Enable the highest e-signature standard QES from home.
```
Comment (betatim) [inline site/content/english/identification.md:184]:
```suggestion
Enable the highest e-signature standard (QES) from home.
```
This is what we currently use "e-signature standard (QES).." so I'd keep using it

File: site/content/english/_index.md [hunk 3]
```diff
@@ -219,9 +219,34 @@ with legal form requirements or high liability risk.
 
 [//]: # (--------------------------------------------------------------------------------------------------------------)
 
-{{< side-by-side width=narrow bottom=16 img-position=left img=aes-for-biz-home img-width=345 img-alt="A large company building with a big cursor." >}}
+{{< content text=center top=4 width=narrow >}}
 {{% markdown %}}
-## Organisation-wide implementation with just a few clicks
+Online identification via video
+or in a few days the whole organization
```
Comment (betatim) [inline site/content/english/_index.md:225]:
```suggestion
or in a few days for your whole organization
```
What do you think?

File: site/content/english/enable-qes.md [hunk 4]
```diff
@@ -8,10 +8,41 @@ slug: enable-qes
 og_image: og/og-skribble-identification.jpg
 
 ---
+{{< content text=center top=16 bottom=8 width=narrow >}}
+{{% markdown %}}
+# Maximise the legal weight of your electronic signature
+Enable the QES by getting yourself identified online via video-call or
+by personal contact with the ID app by Swisscom.
```
Comment (betatim) [inline site/content/english/enable-qes.md:15]: Currently the opening sentence on this page is "Enable the highest e-signing standard (QES) by...". I think this reads better than "Enable the QES...". A normal user probably doesn't know what QES means (do I want this? do I need it?) and it reads like Denglish (in particular the use of "the". I can't explain the grammatical rule, it just sounds weird).

I think by reusing the sentence we already have we can side-step both issues.

File: site/content/english/signaturestandards.md [hunk 5]
```diff
@@ -11,8 +11,8 @@ slug: signaturestandards
 How much legal weight 
 does my e-signature need?
 =========================
-The law distinguishes between qualified e-signatures (QES),
-advanced e-signatures (AES) and simple electronic signature (SES). The three 
+The law distinguishes between the qualified electronic signature (QES), 
+the advanced electronic signature (AES) and the simple electronic signature (SES). The three
```
Comment (betatim) [inline site/content/english/signaturestandards.md:15]: Why are we changing this?

File: site/content/german/enable-qes.md [hunk 6]
```diff
@@ -8,15 +8,43 @@ slug: qes-freischalten
...
-{{< content text=center top=16 width=narrow >}}
+{{< content text=center top=16 bottom=8 width=narrow >}}
 {{% markdown %}}
 Maximiere die Beweiskraft 
 deiner E-Signatur
 ===============
-Schalte den höchsten verfügbaren E-Signatur-Standard (QES) frei,
-in dem du dich im persönlichen Kontakt mit der ID-App
-von Swisscom identifizieren lässt.
+Die (QES) schaltest du frei, in dem du dich online via Video-Call oder im persönlichen Kontakt mit der ID-App von Swisscom identifizieren lässt.
```
Comment (betatim) [inline site/content/german/enable-qes.md:16]: "Die (QES)..." is missing something or has spurious ()

**Comment by michelfernandez** [comment]
Hi Tim, thanks already to check some spellings. 
But I still working on this pull-request. Ist not finished yet. I have to go over everything again today and make some adjustments.

File: site/content/german/_index.md [hunk 7]
```diff
@@ -236,9 +236,33 @@ Behördendokumente
...
+===============
+{{% /markdown %}}
+{{< /content >}}
+
+{{< content top=4 bottom=24 width=narrow display=flex >}}
+{{< step >}}
+{{< picture "video-ident" 250 "Eine lupe vergrößert das bild eines mannes in einem pass." >}}
+{{% markdown %}}
+**Video-Identifikation**
+Schalte die QES von zu Hause aus frei, in dem du dich online via Video-Call oder im persönlichen Kontakt mit der ID-App von Swisscom identifizieren lässt.
```
Comment (Kirastina) [inline site/content/german/_index.md:252]:
```suggestion
Schalte die qualifizierte elektronische Signatur (QES) von zu Hause aus frei, in dem du dich via Video-Call identifizieren lässt.
```

File: site/content/german/_index.md [hunk 8]
```diff
@@ -236,9 +236,33 @@ Behördendokumente
...
+  "Mehr infos"
+  "https://www.videoident.me/"
+  "_blank"
+  "outline">}}
+{{< /step >}}
+
+{{< step >}}
+{{< picture "aes-for-biz-home-id" 280 "Ein grosser Pfeil, wie er eine Organsation, ein Hochhaus anklickt." >}}
+{{% markdown %}}
+**Organisationsweite Einführung mit wenigen Klicks**
```
Comment (Kirastina) [inline site/content/german/_index.md:265]:
```suggestion
**Organisationsweite Einführung**
```

File: site/content/german/_index.md [hunk 9]
```diff
@@ -236,9 +236,33 @@ Behördendokumente
...
 Deine gesamte Mitarbeiterschaft kann innert weniger Tage mit der fortgeschrittenen elektronischen Signatur (FES) signieren.
```
Comment (Kirastina) [inline site/content/german/_index.md:266]:
```suggestion
Schalte die fortgeschrittene elektronische Signatur (FES) innert weniger Tage für die gesamte Mitarbeiterschaft frei.
```

File: site/content/german/_index.md [hunk 10]
```diff
@@ -236,9 +236,33 @@ Behördendokumente
...
+{{< content top=4 bottom=24 width=narrow display=flex >}}
+{{< step >}}
+{{< picture "video-ident" 250 "Eine lupe vergrößert das bild eines mannes in einem pass." >}}
+{{% markdown %}}
+**Video-Identifikation**
+Schalte die QES von zu Hause aus frei, in dem du dich online via Video-Call oder im persönlichen Kontakt mit der ID-App von Swisscom identifizieren lässt.
+{{% /markdown %}}
+<br>
+{{< button
+  "Mehr infos"
```
Comment (Kirastina) [inline site/content/german/_index.md:256]:
```suggestion
  "Mehr erfahren"
```

File: site/content/english/_index.md [hunk 11]
```diff
@@ -219,9 +219,34 @@ with legal form requirements or high liability risk.
...
+===============
+{{% /markdown %}}
+{{< /content >}}
+
+{{< content top=4 bottom=24 width=narrow display=flex >}}
+{{< step >}}
+{{< picture "video-ident" 250 "A magnifying glass enlarges the image of a man in a passport." >}}
+{{% markdown %}}
+**Online identification**
+Enable the QES by getting yourself identified online via video-call or
```
Comment (Kirastina) [inline site/content/english/_index.md:235]:
```suggestion
Enable the qualified electronic signature (QES) from home by getting yourself identified via video-call.
```

File: site/content/french/_index.md [hunk 12]
```diff
@@ -215,9 +215,33 @@ avec exigences formelles juridiques ou avec un risque de responsabilité fort.
...
+===============
+{{% /markdown %}}
+{{< /content >}}
+
+{{< content top=4 bottom=24 width=narrow display=flex >}}
+{{< step >}}
+{{< picture "video-ident" 250 "Une loupe agrandit l'image d'un homme dans un passeport." >}}
+{{% markdown %}}
+**Identification vidéo**
+Déverrouillez la norme de signature électronique la plus élevée (SEQ) en vous identifiant en ligne par appel vidéo ou par contact personnel avec l’ID-app de Swisscom.
```
Comment (Kirastina) [inline site/content/french/_index.md:231]:
```suggestion
Déverrouillez la signature électronique qualifiée (SEQ) depuis votre domicile en vous identifiant en ligne par appel vidéo.
```

File: site/content/english/_index.md [hunk 13]
```diff
@@ -219,9 +219,34 @@ with legal form requirements or high liability risk.
...
+  "Learn more"
+  "https://www.videoident.me/"
+  "_blank"
+  "outline">}}
+{{< /step >}}
+
+{{< step >}}
+{{< picture "aes-for-biz-home-id" 280 "A big arrow, clicking on an organization, a high-rise building." >}}
+{{% markdown %}}
+**Organisation-wide implementation with just a few clicks**
```
Comment (Kirastina) [inline site/content/english/_index.md:249]:
```suggestion
**Organisation-wide implementation**
```

File: site/content/english/_index.md [hunk 14]
```diff
@@ -219,9 +219,34 @@ with legal form requirements or high liability risk.
...
 Enable your entire staff to sign with the advanced electronic signature (AES) within just a few days.
```
Comment (Kirastina) [inline site/content/english/_index.md:250]:
```suggestion
Enable the advanced electronic signature (AES) for your entire staff within just a few days.
```

File: site/content/french/_index.md [hunk 15]
```diff
@@ -215,9 +215,33 @@ avec exigences formelles juridiques ou avec un risque de responsabilité fort.
...
+  "En savoir plus"
+  "https://www.videoident.me/"
+  "_blank"
+  "outline">}}
+{{< /step >}}
+
+{{< step >}}
+{{< picture "aes-for-biz-home-id" 280 "Une grande flèche, un clic sur une organisation, un immeuble de grande hauteur." >}}
+{{% markdown %}}
+**Implémentation à toute l'organisation en quelques clics**
```
Comment (Kirastina) [inline site/content/french/_index.md:244]:
```suggestion
**Implémentation à toute l'organisation**
```

File: site/content/french/_index.md [hunk 16]
```diff
@@ -215,9 +215,33 @@ avec exigences formelles juridiques ou avec un risque de responsabilité fort.
...
 Permettez à l'ensemble de votre personnel de signer avec la signature électronique avancée (SEA) en quelques jours seulement.
```
Comment (Kirastina) [inline site/content/french/_index.md:245]:
```suggestion
Déverrouillez la signature électronique avancée (SEA) pour l'ensemble de votre personnel en quelques jours seulement.
```

File: site/content/german/enable-qes.md (hunk 6 above)
Comment (michelfernandez) [reply site/content/german/enable-qes.md]: Yes, thank you very much! Have I corrected in DE and FR :-)

File: site/content/english/signaturestandards.md (hunk 5 above)
Comment (michelfernandez) [reply site/content/english/signaturestandards.md]: I've adjusted.

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I found some stuff that will be fixed later on. This was discussed with @michelfernandez and @fursinger  I will create a bugtask after I tested https://github.com/BlockSigner/wave/pull/889 and will link it in the next comment.

**Comment by michelfernandez** [comment]
Talked to Philipp and Patrick yesterday and we can put this stand online with the video-ident. Since this pull-request is fully tested, I will remove in a separate pull-request notice that you can make identifications in the Flex-Office as soon as it is merged.

**Comment by fursinger** [comment]
I look through the changed pages and obviously, it's too many changes to check again in detail. LGTM.

**Pull request merged**